docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -10,6 +10,88 @@ which conforms to the schema defined below. You may find it easier to
 start writing your ``DOCUMENTATION`` string in an editor with YAML
 syntax highlighting before you include it in your Python file.
 
+All modules must have the following sections defined in this order:
+
+1. ANSIBLE_METADATA
+2. DOCUMENTATION
+3. EXAMPLES
+4. RETURNS
+5. Python imports
```

Comment:
I've added a TOC in now, so that should make it easier for people to navigate the page